staging/src/k8s.io/dynamic-resource-allocation/controller/controller.go

```diff
@@ -500,7 +501,14 @@ func (ctrl *controller) syncClaim(ctx context.Context, claim *resourcev1alpha2.R
 		return err
 	}
 
-	return ctrl.allocateClaim(ctx, claim, claimParameters, class, classParameters, "", nil)
+	claimAllocations := claimAllocations{&ClaimAllocation{
+		Claim:           claim,
+		ClaimParameters: claimParameters,
+		Class:           class,
+		ClassParameters: classParameters,
+	}}
```

Comment:
Huh? Is this PR still WIP?

It looks like `ctrl.allocateClaims` is always called for exactly one claim, so where is batching happening?

Reply:
Never mind. When handling PodSchedulingContext below, it gets called with more than one claim.